### PR TITLE
add input file example with 1M line

### PR DESCRIPTION
**Pull request by aboullaite**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +73,8 @@ Assist Adnan by writing a program, using any allowed language, to find the city
 
 ### INPUT
 
+You can find an example of the input file [here](./input.txt). This file contains 1 million line to run your tests and validate your solution.
```
Comment (aboullaite) [inline README.md:76]: @chermehdi What is the expected file location ? same for output file if any

Comment (chermehdi) [reply README.md:76]: Output should be a file named `output.txt` in the working directory of the process

Comment (chermehdi) [inline README.md:76]: Also, would've been nice to give people the expected output as well.

File: README.md [hunk 2]
```diff
@@ -39,12 +39,14 @@ Assist Adnan by writing a program, using any allowed language, to find the city
...
 
-- 1.00 <= price <= 10.00
+- 1.00 <= price <= 100.00
 - we have 101 cities.
 - We have 100 products.
 - Each city may have the same product duplicated several times with different or identical prices.
 - Your code will process a 10^9 (1 billion) line CSV file.
 - You are allowed to code with any language from this list: C, C++, Java, Python, Rust, JavaScript, PHP.
+- The input file is located on "input.txt" 
+- The output file should be located in the same fodler directory as your submission `./output.txt`
```
Comment (chermehdi) [inline README.md:49]: @aboullaite what does this mean?

the output file should mean where the output file will be written by the solution, and it should write the result to a file called `output.txt`. This makes it sound that you have to submit the output file as well, which I don't think what we want

**Review by chermehdi — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (aboullaite) [reply README.md:49]: How about ?
```suggestion
- Make sure the result are saved in a file saved under `./output.txt`
```

File: README.md (hunk 1 above)
Comment (aboullaite) [reply README.md:77]: Once we agreed on the program I can run it and share the output for the 1M rows

File: README.md (hunk 2 above)
Comment (chermehdi) [reply README.md:49]: yeah looks better (we can even provide a dummy solution that reads from `input.txt` and prints to `output.txt`)

**Review by chermehdi — APPROVED** [review]
Thanks looks good

**Pull request merged**